test/common/http/http2/http2_frame.cc

```diff
@@ -51,12 +51,22 @@ Http2Frame::ResponseStatus Http2Frame::responseStatus() const {
     return ResponseStatus::Ok;
   case StaticHeaderIndex::Status404:
     return ResponseStatus::NotFound;
+  case StaticHeaderIndex::Status500:
+    return ResponseStatus::InternalServerError;
   default:
     break;
   }
   return ResponseStatus::Unknown;
 }
 
+uint32_t Http2Frame::streamIndex() const {
```

Comment:
nit: I think we typically refer to this as stream id, not index.

Reply:
Done